controllers/userController.js

```diff
@@ -0,0 +1,34 @@
+const { model } = require('mongoose')
```

Comment:
First line should be const mongoose = require('mongoose')

Reply:
removed mongoose from controller